### ISSUE TITLE
Add warning for IE 9 and 10

### ISSUE THREAD

**Issue by SilentGert** [effektif/admin-docs#35]
The support for IE 9 and 10 will be dropped by the end of 2017. The list of supported browsers should be updated.

**Comment by SilentGert** [comment]
@TimKam Did you plan on merging the changes on this branch?

**Comment by TimKam** [comment]
Yes, but there's probably a "false positive" test and I need to see how to fix this.
